src/Common/MiniTerminal.cpp

```diff
@@ -202,42 +187,26 @@ void MiniTerminal::writeError(const char* result)
 
 void MiniTerminal::executeCommand(const char* cmdLine)
 {
-    if (0 == strcmp(cmdLine, PING))
-    {
-        cmdPing(&cmdLine[PING_LEN]);
-    }
-    else if (0 == strcmp(cmdLine, RESET))
-    {
-        cmdReset(&cmdLine[RESET_LEN]);
-    }
-    else if (0 == strncmp(cmdLine, WRITE_WIFI_PASSPHRASE, WRITE_WIFI_PASSPHRASE_LEN))
-    {
-        cmdWriteWifiPassphrase(&cmdLine[WRITE_WIFI_PASSPHRASE_LEN]);
-    }
-    else if (0 == strncmp(cmdLine, WRITE_WIFI_SSID, WRITE_WIFI_SSID_LEN))
-    {
-        cmdWriteWifiSSID(&cmdLine[WRITE_WIFI_SSID_LEN]);
-    }
-    else if (0 == strncmp(cmdLine, GET_IP, GET_IP_LEN))
-    {
-        cmdGetIPAddress(&cmdLine[GET_IP_LEN]);
-    }
-    else if (0 == strncmp(cmdLine, GET_STATUS, GET_STATUS_LEN))
+    uint32_t idx = 0U;
+
+    for (idx = 0U; UTIL_ARRAY_NUM(m_cmdTable) > idx; ++idx)
     {
-        cmdGetStatus(&cmdLine[GET_STATUS_LEN]);
+        const CmdTableEntry entry = m_cmdTable[idx];
```

Comment:
Using a reference might be better than copy.

Reply:
I acutally did this before pushing and noticed that codesize did surprisingly increase by 20 bytes again. So I left it as a copy. It is just two 32bit values. I guess they load into registers and are cheaper to handle compared to a reference  deref.